src/Resolvers/Microsoft.DotNet.NativeWrapper/NETCoreSdkResolverNativeWrapper.cs

```diff
@@ -50,5 +50,32 @@ public static string[] GetAvailableSdks(string dotnetExeDirectory)
 
             return list.Entries;
         }
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal struct hostfxr_initialize_parameters
+        {
+
+        }
```

Comment:
This would be invalid if used - the structure must have at least the first `size` field - it's used effectively as a versioning mechanism. And it MUST be set before calling the initialize function.
https://github.com/dotnet/runtime/blob/a79c62ddc8089cf2879ed36eac9aa333b32bde5f/src/native/corehost/hostfxr.h#L78

Reply:
This also still needs to be done.